test/test_helper.rb

```diff
@@ -314,7 +314,7 @@ def select2(value, attrs)
     end
 
     def wait_for_ajax
-      Timeout.timeout(Capybara.default_wait_time) do
+      Timeout.timeout(Capybara.default_max_wait_time) do
```

Comment:
We have some [additional](https://github.com/theforeman/foreman/blob/develop/test/integration/host_test.rb#L270) places that use `Capybara.default_wait_time`. Can you change them too?